ci-scripts/Jenkinsfile-Magma-Sanity-Check

```diff
@@ -141,6 +141,54 @@ pipeline {
               echo "Upstream Job passed SPGWU_BRANCH to use: ${spgwuBranch}"
             }
           }
+
+          // Prepare workspace
+          sh './scripts/syncComponents.sh --hss-branch ' + hssBranch + ' --spgwc-branch ' + spgwcBranch + ' --spgwu-tiny-branch ' + spgwuBranch
+
+          // On Ubuntu servers, we shall pull from private local registry
+          if ((cn_ci_host == 'selfix') || (cn_ci_host == 'orion')) {
+            // Login
+            sh containerCli + ' login -u oaicicd -p oaicicd selfix.sboai.cs.eurecom.fr > /dev/null 2>&1'
```

Comment:
@rdefosse There is no possibility to mask the password or save it as Jenkins secret?